maintainers/maintainer-list.nix

```diff
@@ -11719,6 +11719,12 @@
     github = "mikefaille";
     githubId = 978196;
     name = "Michaël Faille";
+  };
+    mikecm = {
+    email = "mikecmcleod@gmail.com";
+    github = "MaxwellDupre";
+    githubId = 14096356;
+    name = "Michael McLeod";
```

Comment:
Could you please split this change to the separate commit, `maintainers: add mikecm`?
Also, it should be sorted by alphabetical order.